### ISSUE TITLE
How to launch multimode training on 2 A100 machines through hugging face

### ISSUE THREAD

**Issue by Shashi456** [huggingface/accelerate#617]
### System Info

```Shell
Accelerate version: Version: 0.13.0.dev0
Accelerate config: - 
OS: Ubuntu VERSION="20.04.4 LTS (Focal Fossa)"
Torch version: Version: 1.12.1+cu116
numpy version: Version: 1.21.0
```

### Information

- [ ] The official example scripts
- [X] My own modified scripts

### Tasks

- [ ] One of the scripts in the examples/ folder of Accelerate or an officially supported `no_trainer` script in the `examples` folder of the `transformers` repo (such as `run_no_trainer_glue.py`)
- [X] My own task or dataset (give details below)

### Reproduction

Trying to run the model for evaluation for the big science project

CODE: https://github.com/bigscience-workshop/lm-evaluation-harness

While running the model we pass the flags as ` --model_args pretrained=bigscience/bloom,use_accelerate=True,dtype=bfloat16` and we wrote the accelerate integration [here](https://github.com/bigscience-workshop/lm-evaluation-harness/blob/87ef365476b4d495673e3ec0d18afc222a5091a3/lm_eval/models/huggingface.py#L15)

I have a 2x A100 with 8GPUs each, and the ssh sits at /job/hostfile, how could I get this running on a multinode?

### Expected behavior

```Shell
Run on both the gpus.
```

**Comment by Shashi456** [comment]
Not a bug sorry, i did not add the label.

**Comment by sgugger** [comment]
Please use the [forums](https://discuss.huggingface.co/) for such questions as we keep the issues for bugs and feature requests only 🤗 

You should run `accelerate config` on both machines and answer the questionnaire, then `accelerate launch path_to_script --args` on both machines. You can also directly use the PyTorch launcher if you are familiar with it.

**Comment by Shashi456** [comment]
Thanks a lot @sgugger sorry for placing the issue here.
